documentation/code-gen/configure-python-sdk.md

```diff
@@ -142,10 +142,7 @@ python:
 no-namespace-folders: true
 output-folder: $(python-sdks-folder)/appconfiguration/azure-mgmt-appconfiguration/azure/mgmt/appconfiguration
 ```
-``` yaml $(python-mode) == 'create'
-python:
-  basic-setup-py: true
-  output-folder: $(python-sdks-folder)/appconfiguration/azure-mgmt-appconfiguration
+``` yaml $(python-mode) == 'create' && $(track2)
```

Comment:
How about this? https://github.com/Azure/azure-rest-api-specs/pull/11542/files#diff-694a0a1176057a399f20a5338d48566e5501d2c8d34e57f82a6d1272c28114d3L135-L144

Reply:
yes, committed